### DISCUSSION TITLE
Configure to skip some versions

### DISCUSSION THREAD

**Discussion by ajax-kovtunov-o** [renovatebot/renovate#29013] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

github action

### Please tell us more about your question or problem

My configuration

```json
"packageRules": [
    {
      "matchDatasources": ["maven", "gradle"],
      "managers": [
        "gradle"
      ],
      "registryUrls": [
        "s3://mycompany-maven-repository/snapshots"
      ],
      "automerge": false,
      "allowedVersions": "^[a-fA-F0-9]+\\.MASTER-SNAPSHOT$"
    }
    ]
}
```

The problem is 

There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.

Location: config
Error type: Invalid allowedVersions
Message: The following allowedVersions does not parse as a valid version or range: "^[a-fA-F0-9]+\.MASTER-SNAPSHOT$"

Why do I need this behavia?
Renovate opens PRs with versions like this `2.5.2.139.9f9fe3b-SNAPSHOT`, but I need to use only `*.MASTER-SNAPSHOT` versions.

### Logs (if relevant)

<details><summary>Logs</summary>

```

There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.

Location: config
Error type: Invalid allowedVersions
Message: The following allowedVersions does not parse as a valid version or range: "^[a-fA-F0-9]+\.MASTER-SNAPSHOT$"

```

</details>

**Comment by rarkins** [comment, marked as answer]
You need to start and end your Regex with /

**Reply by HonkingGoose** [reply]
I read past this key point myself when reading the `allowedVersions` docs... :upside_down_face: So I'm improving the docs with this PR:

- #29014

**Reply by rarkins** [reply]
I think the old text was very clear too
